Automattic/wp-calypso
# Add atomic hosting eligibility check to limit rollout to new sites

#### Changes proposed in this Pull Request

* Adds a selector which limits atomic hosting feature to new sites

#### Testing instructions

* Checkout this PR and check an Atomic site with an ID > 168768859 and the hosting menu options should still appear
* Check again with an ID < 168768859 and their should be no hosting menu options

Implements UI part of  #37236
